bread/layout/components/tooltip.py

```diff
@@ -0,0 +1,269 @@
+from typing import Optional, Union
+
+import htmlgenerator as hg
+
+from bread.layout.components.button import Button
+from bread.layout.components.icon import Icon
+from bread.utils import Link
+
+
+def _get_icon(icon):
+    """Return a proper Icon instance"""
+    if isinstance(icon, str):
+        icon = Icon(icon, size="16")
+    else:
+        icon.attributes["_class"] = icon.attributes.get("_class", "") + " bx--btn__icon"
```

Comment:
Sorry, I forgot to mention this one before: htmlgenerator.lazy objects do currently not support the plus operator. Therefore we have to write it like:

```hg.BaseElement(icon.attributes.get("_class", ""), " bx--btn__icon")```

We will add support for expression for htmlgenerator.Lazy in the future, but not yet, so we need to wrap this with a BaseElement.

Reply:
Thank you for the information. Though I later discovered there's no need to add the class for that one.